crossbar/controller/node.py

```diff
@@ -415,9 +415,12 @@ def _startup(self, config):
                     # we've established a connection to the component
                     def component_exited(info):
                         dead_comp = info['id']
-                        self.log.info("Component '{}' failed to start; shutting down node.".format(dead_comp))
-                        if self._reactor.running:
+                        self.log.info("Component '{dead_comp}' failed to start; shutting down node.", dead_comp=dead_comp)
```

Comment:
minor thing, you could pass in info and iirc use "{info.id}" here and "{info.config}" in the formatting string. I need to do this more often when I touch things...